pages/foods/[id].js

```diff
@@ -13,14 +13,20 @@ export default function FoodDetailsPage({
   handleUpdateFood,
   handleRemoveGoodCat,
   handleRemoveBadCat,
-  isEditingGood,
-  setIsEditingGood,
-  isEditingBad,
-  setIsEditingBad,
+  isEditingGoodCat,
+  setIsEditingGoodCat,
+  isEditingBadCat,
+  setIsEditingBadCat,
 }) {
   const router = useRouter();
   const { id } = router.query;
 
+  // two qay link (from the internet)
+  const goBack = (e) => {
```

Comment:
it`s just in my opinion but i would not use (e) as function argument, because it reduces the readability of your code, but this is just my opinion.

Reply:
changed it to "event" :)